src/main/resources/templates/notification/publisher/mattermost.peb

```diff
@@ -0,0 +1,5 @@
+{
+    "username": "Dependency Track",
```

Comment:
Should this be configurable? Will a user with name `Dependency Track` have to exist, or is this just a display name and can be arbitrary? I'm not familiar with Mattermost.

Reply:
This is just a display name. We could also not define it at all and let the webhook configure it.